andela/mrm_api
# #162815645 Remove alembic multiple head error

#### What does this PR do?
Eliminates two or more versions that refer to the same parent migration as their ancestor.
Ensure only a single head is supported serially.
#### Description of Task to be completed?
To fix it you just have to adjust the `down_revision` of your migration to be that of the latest one. This has been achieved by manually editing the revision files and deleting the unnecessary ones.
#### How should this be manually tested?
Clone the branch and run `alembic stamp head` followed with `alembic upgrade head` in the terminal.
Run `alembic history` in the terminal to see the new changes.
#### Any background context you want to provide?
Not applicable
#### What are the relevant pivotal tracker stories?
#162815645
